R/events.R

```diff
@@ -166,6 +168,14 @@ attach_event_listeners <- function(
     individual::update_category_listener(variables$mosquito_state, 'Im')
   )
 
+  events$mosquito_death$add_listener(
+    function(timestep, target) {
+      variables$mosquito_state$queue_update('Unborn', target)
```

Comment:
Minor point - but (speaking with my entomologist hat on) its not clear to me what "Unborn" as a mosquito state refers to. It could mean an individual that doesn't exist, an egg, a larvae. a pupae... Perhaps we can make it a bit more clear?